text/0074-structured-vcd.md

```diff
@@ -112,17 +112,34 @@ These images demonstrate the difference in GTKWave output:
 
 Currently, Amaranth represents aggregate signals in the VCD by appending the names of elements/members to the name of the signal.
 When simulator output is written to a VCD file, we intend that signals with aggregate datatypes are explicitly given their own scope and split into multiple VCD variables.
+
+### Changes to VCD Writing
+
 We propose the following conventions:
 
 - The `module` scope defines a group of signals belonging to the same `Module`
 - The `vhdl_array` scope defines a group of signals belonging to an array (such as a signal with `ArrayLayout`)
-- The `vhdl_record` scope defines a structured group of signals (such as a signal with `StructLayout`)
+- The `vhdl_record` scope defines a structured group of signals (such as a signal with `StructLayout`, `UnionLayout`, or `FlexibleLayout`)
+
+After these changes, this functionality will be enabled by default in the VCD writer.
+These changes will **not** be backported to Amaranth 0.5.x.
+Users that require compatibility with the pre-RFC behavior are expected to either:
+
+- Opt-out on a case-by-case basis by passing arguments to `write_vcd()` (ie. `structured=False`)
```

Comment:
I suspect we may be using other VCD extensions later (in fact, we are using at least one already: strings; this is what's crashing the Playground currently).

How about `pure=True` and `AMARANTH_PURE_VCD=1`, to reflect the part that we deviate from the official VCD definition in IEEE 1800?

Reply:
Ooh, that's a good point - sounds great.